.travis.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2018 Shapelets.io
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+language: python
```

Comment:
This may not be need as you are running in docker

Reply:
At the end, I am using python, I have changes the way I upload the coverage in order to not to publish the token.